doc/build/orm/dataclasses.rst

```diff
@@ -37,8 +37,8 @@ as having Dataclass-specific behaviors, most notably  by taking advantage of :pe
 as though it were explicitly decorated using the ``@dataclasses.dataclass``
 decorator.
 
-.. note::  Support for :pep:`681` in typing tools as of **July 3, 2022** is
-   limited and is currently known to be supported by Pyright_, but not yet
+.. note::  Support for :pep:`681` in typing tools as of **March 6, 2023** is
+   limited and is currently known to be fully supported by Pyright_ and
```

Comment:
I would take the word "limited" out , at this point

Reply:
indeed. 

>  as of **March 9, 2023** is currently known to be fully supported

(btw today it's the 9th :)

Reply:
https://github.com/sqlalchemy/sqlalchemy/pull/9463/commits/27ecc8321f0a6298b9bd7a56ac059c087d6247bc

Reply:
> indeed.
> 
> > as of **March 9, 2023** is currently known to be fully supported
> 
> (btw today it's the 9th :)

I've added 6th as it was the release date of `mypy 1.1.1` ;), but I could change to today as well

Reply:
either one is fine. we could also use just the month. It's there just to give a general feel for time when the statement was written/updated

Reply:
we can let this poor submitter go free, it's fine